openmmtools/tests/test_sampling.py

```diff
@@ -161,9 +161,35 @@ def run(self, include_unsampled_states=False):
             logger.setLevel(logging.CRITICAL)
             simulation.run()
 
-            # Create Analyzer.
+            # Create Analyzer specfiying statistical_inefficiency without n_equilibration_iterations and 
+            # check that it throws an exception
+            assert_raises(Exception, self.ANALYZER, reporter, statistical_inefficiency=10)
```

Comment:
Not blocking. Just to have a better readability of what the test is doing here I suggest the following change 
```suggestion
            with assert_raises(Exception):
                self.ANALYZER(reporter, statistical_inefficiency=10)
```

Reply:
Marked as solved, since it was merged during the re-review.